src/lib/EventProvider.js

```diff
@@ -10,13 +10,14 @@ class EventProvider {
     this._eventPublisher = eventPublisher;
   }
 
-  onNewEvent = (callback: (event: Event) => void) => {
+  onNewEvent = (callback: (event: Event) => void, eventNamePrefix: string = '*') => {
     this._eventPublisher.subscribe(
-      '*',
+      eventNamePrefix,
       this._onNewEvent(callback),
       {
         filterOptions: {
           listenToBroadcastedEvents: false,
+          listenToIPC: false,
```

Comment:
Beside Debug there is no meaningfull workflow possible, cause this is an Response for an action from other Code (maybe in another process). So if someone removes such an action, and another one is deppending on the internal request or response, -> totally mess

I would also suggest to seperate the EventEmitter in two different Instances
* events
* internal events

to make this more clear